cloud-config/main.yml

```diff
@@ -118,6 +118,10 @@ vm_extensions:
   cloud_properties:
     elbs:
     - (( grab terraform_outputs.shibboleth_elb_name ))
+- name: shibboleth-target
```

Comment:
Why is this a new extension? shouldn't the `lb_target_groups` be tacked on to the existing `shibboleth-lb` extension like the other config changes?